docs/sources/k6/next/using-k6/thresholds.md

```diff
@@ -35,7 +35,7 @@ The other evaluates whether 95 percent of responses happen within a certain dura
 
 {{< code >}}
 
-<!-- md-k6:skip -->
```

Comment:
Many in this file need to be run without thresholds because otherwise they will fail due to network latency variations.

Reply:
However at least now we are not skipping them.

Reply:
That's nice, thanks for adding this option! 🙇